atlasdb-ete-tests/src/test/java/com/palantir/atlasdb/ete/LockWithoutTimelockEteTest.java

```diff
@@ -1,37 +0,0 @@
-/*
- * (c) Copyright 2019 Palantir Technologies Inc. All rights reserved.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package com.palantir.atlasdb.ete;
-
-import static org.assertj.core.api.Assertions.assertThat;
-
-import org.junit.Test;
-
-import com.palantir.atlasdb.lock.LockResource;
-
-public class LockWithoutTimelockEteTest {
-    private LockResource lockResource = EteSetup.createClientToSingleNode(LockResource.class);
-
-    @Test
-    public void hugeV1LockSucceeds() throws InterruptedException {
-        assertThat(lockResource.lockUsingLegacyLockApi(100, 500_000)).isTrue();
```

Comment:
Let's talk about this tomorrow. I don't see how this change would affect behaviour here, but because of large internal product's lock service implementation, we do need to know if the huge requests were broken.

Reply:
Reverted the change as discussed